### ISSUE TITLE
Hyperparameters for training RGB/MS restoration model

### ISSUE THREAD

**Issue by haydenhs** [cindydeng1991/TPAMI-CU-Net#3]
Hi Cindy, thanks for sharing your code.

I'm working on the RGB/MS MIR task using Columia multi-spectral database (CAVE) according to your paper. However I tried a few combinations of different hyperparameters and still failed to train a satisfactory model. Could you please share your training hyperparameters (e.g. `learning rate, filter size, num_of_layers...`)for RGB/MS restoration task?

Best regards,
hayden
